csss-site/src/events/templates/events/fall_hacks2020.html

```diff
@@ -0,0 +1,26 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>
+
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+
+
+ 
+
+      <header>
+	      <h1> <a href="{{URL_ROOT}}events/fall_hacks_submissions2020">Project Page of submissions from the Computing Science Student Society Fall Hacks 2020 submissions<a></h1>

```

Comment:
bit redundant much, no?
why not `Computing Science Student Society Fall Hacks 2020 submissions`?

also, why does a link need a header tag?